### ISSUE TITLE
Masked LM pre-training

### ISSUE THREAD

**Issue by Khedesh** [CyberZHG/keras-bert#219]
Hello,

I want to train BERT from scratch by training on Masked LM and NSP tasks.
Training on NSP task is straightforward, but, for training on Masked LM I get confused.

Can you help me by describing how to run Masked LM using keras-bert?

Regards,
Khedesh
